src/ophyd_epics_devices/zebra.py

```diff
@@ -0,0 +1,302 @@
+            ],
+        }
+
+        super().__init__(prefix, name)
+
+    async def apply_logic_gate_config(
+        self, type: GateType, gate_number: int, config: LogicGateConfiguration
+    ):
+        """Uses the specified `LogicGateConfiguration` to configure a gate on the Zebra.
+        Args:
+            type (GateType): The type of gate e.g. AND/OR
+            gate_number (int): Which gate to configure.
+            config (LogicGateConfiguration): A configuration for the gate.
+        """
+        gate: GateControl = self.all_gates[type][gate_number - 1]
+
+        await gate.enable.set(boolean_array_to_integer([True] * len(config.sources)))
+
+        # Input Source
+        for source_number, source_pv in enumerate(gate.sources_list):
+            try:
+                await source_pv.set(config.sources[source_number])
+            except IndexError:
+                await source_pv.set(self.DEFAULT_SOURCE_IF_GATE_NOT_USED)
+
+        # Invert
+        await gate.invert.set(boolean_array_to_integer(config.invert))
+
+    async def reset(self):
+        ...
+
+    apply_and_gate_config = partialmethod(apply_logic_gate_config, GateType.AND)
+    apply_or_gate_config = partialmethod(apply_logic_gate_config, GateType.OR)
+
+
+class PositionCompare(StandardReadable):
+    def __init__(self, prefix: str, name="") -> None:
+        self.capture = Capture("PC_BIT_CAP:")
+        self.array_outputs = ArrayOuts("")
+
+        self.direction = EpicsSignalRW(Direction, "PC_DIR")
+        self.time_units = EpicsSignalRW(TimeUnits, "PC_TSPRE")
+        self.time = EpicsSignalRW(float, "PC_TIME")
+
+        self.pulse_sel = EpicsSignalRW(GatePulseSelect, "PC_PULSE_SEL")
+        self.pulse_input = EpicsSignalRW(int, "PC_PULSE_INP")
+        self.pulse_start = EpicsSignalRW(float, "PC_PULSE_START")
+        self.pulse_width = EpicsSignalRW(float, "PC_PULSE_WID")
+        self.pulse_delay = EpicsSignalRW(int, "PC_PULSE_DLY")
+        self.pulse_step = EpicsSignalRW(int, "PC_PULSE_STEP")
+        self.pulse_max = EpicsSignalRW(int, "PC_PULSE_MAX")
+
+        self.num_gates = EpicsSignalRW(int, "PC_GATE_NGATE")
+        self.gate_trigger = EpicsSignalRW(EncoderType, "PC_ENC")
+        self.gate_sel = EpicsSignalRW(GatePulseSelect, "PC_GATE_SEL")
+        self.gate_input = EpicsSignalRW(int, "PC_GATE_INP")
+        self.gate_start = EpicsSignalRW(float, "PC_GATE_START")
+        self.gate_width = EpicsSignalRW(float, "PC_GATE_WID")
+        self.gate_step = EpicsSignalRW(int, "PC_GATE_STEP")
+
+        self.arm_sel = EpicsSignalRW(ArmSelect, "PC_ARM_SEL")
+        self.arm = EpicsSignalRW(int, "PC_ARM")
+        self.disarm = EpicsSignalRW(int, "PC_DISARM")
+        self.armed = EpicsSignalRW(int, "PC_ARM_OUT")
+
+        self.captured = EpicsSignalR(int, "PC_NUM_CAP")
+        self.downloaded = EpicsSignalR(int, "PC_NUM_DOWN")
+        super().__init__(prefix, name)
+
+        # want a mapping between the encoder, capture and array pvs?
```

Comment:
Should: This comment doesn't add anything to the code. If it's something that needs doing outside the scope of this we should make a new issue and remove the comment